app/assets/stylesheets/_form.scss

```diff
@@ -1,99 +1,115 @@
-								color: green;
-							}
-							input[type="submit"]:hover{
-								opacity: 0.9;
-								cursor: pointer;
-							}
-							.decision{
-								padding: 0;
-								input{
-									width: 100%;
-									margin-bottom: 0.5em;
-								}
-							}
+  body{
+    .wrapper{
+      article{
+        section{
+          div{
+            form{
+              padding-top: 1em;
+              label{
+                display: block;
+                font-weight: 300;
+                font-size: 1em;
+                margin-bottom: 0.5em;
+              }
+              input[type="text"],input[type="email"],input[type="password"],textarea{
+                margin-bottom: 0.5em;
+                padding: 1em 0;
+                text-indent: 1em;
+                width: 100%;
+                display: block;
+                border-style: none;
+                @include border-radius(0.25em);
+                box-shadow:inset 0 0 0.125em 0 $map_blue;
+                font-size: 0.85em;
+                text-align: left;
+              }
+              textarea{
+                resize: none;
+                padding: 1em;
+                text-indent: 0;
+                box-sizing: border-box;
+                height: 10em;
+              }
+              input[type="submit"]{
+                display: block;
+                width: 40%;
+                margin: 0.25em 0;
+                text-align: center;
+                border-style: none;
+                @include border-radius(0.25em);
+                background-color: $map_blue;
+                color: white;
+                font-size: 1em;
+                height: 2.5em;
+                -webkit-transition: opacity .25s ease-in-out;
+                   -moz-transition: opacity .25s ease-in-out;
+                        transition: opacity .25s ease-in-out;
+              }
+              input[type="submit"].recommended{
+                background-color: $light_green;
+                box-shadow:inset 0 0 0.125em 0 green;
+                color: green;
+              }
+              input[type="submit"]:hover{
+                opacity: 0.9;
+                cursor: pointer;
+              }
+              .decision{
+                padding: 0;
+                input{
+                  width: 100%;
+                  margin-bottom: 0.5em;
+                }
+              }
 
-							a{
-								text-decoration: none;
-							}
-						}
-					}
-				}
-			}
-		}
-	}
-	
+              a{
+                text-decoration: none;
+              }
+            }
+          }
+        }
+        section.options{
+          padding: 0;
+          background-color: $geometry_gray;
+          div{
+            margin: 0 auto;
+            padding: 0;
+            text-align: left;
+            p{
+              a{
+                color: $map_blue;
+              }
+              a:hover{
+                font-weight: 700;
+              }
+            }
```

Comment:
Style in `section.option` would be the ones I defined. It is kind like ones defined `_hero.scss` but not exactly. It has different width, padding for `div` and text-alignment and padding for `section.option` than that of `hero`, hence inheriting didn't seem like  great idea.